shared/domain/src/main/java/com/ivy/domain/usecase/account/AccountBalanceUseCase.kt

```diff
@@ -33,11 +39,42 @@ class AccountBalanceUseCase @Inject constructor(
     suspend fun calculate(
         account: AccountId,
     ): Map<AssetCode, NonZeroDouble> {
-        TODO("Not implemented")
+
+        val transactions = transactionRepository.findAll()
```

Comment:
inline the variable and make the below line call on 2 lines, will be easier to read IMO